taxi/views.py

```diff
@@ -1,3 +1,17 @@
 from django.shortcuts import render
 
-# Create your views here.
+from taxi.models import Driver, Car, Manufacturer
+
+
+def index(request):
+    num_drivers = Driver.objects.count()
+    num_manufacturers = Manufacturer.objects.count()
+    num_cars = Car.objects.count()
+
```

Comment:
It is not essential to create additional variables here, u can get drivers, manufacturers,cars directly in the context.